### DISCUSSION TITLE
tessellation of Ti-6Al-4V alloy

### DISCUSSION THREAD

**Discussion by my-name-iz-ayush** [neperfepx/neper#704] [Q&A] [answered]
hi @rquey thank you for such a great simulation software. I'm currently working on tessellation of Ti-6Al-4V alloy's microstructure having lamellar microstructure with  2 phases out of which one is having 30% volume fraction for which I used following code:
```
neper -T -n 500::from_morpho -morpho "gg::lamellar(w=0.1)" -domain "cube(3,3,3)" -group "id<=30?1:2" -o Ti64 
```
for this I'm getting following visualization:
```
neper -V Ti64.tess -datacellcol group -datacelltrs 0 -print Ti64
```
![ti64new2](https://github.com/neperfepx/neper/assets/78230636/ac0bcd00-9b3e-4773-a349-097d3c6c5c2a)
![ti64new22](https://github.com/neperfepx/neper/assets/78230636/e5fbe501-e856-42e0-bae4-1b24f73d6f37)

I have to assign two different properties of these two different phases in Abaqus CAE how can I proceed ? I'm taking input in Abaqus using inp file format after meshing the microstructure. Also can you suggest is this tessellation code correct for simulation of Ti-6Al-4V alloy's microstructure or any changes if required.

**Comment by rquey** [comment, marked as answer]
Neper does not write the group information to the inp file, but you can print it to separate file using `-statcell group` in `-T` or `-statelset group` in `-M`. What to do in the inp file then is more an Abaqus question, and I'm afraid I can't help.

**Comment by my-name-iz-ayush** [comment]
Thank you for your support. I will try to look for abaqus.

**Comment by my-name-iz-ayush** [comment]
@rquey can you help me how to mesh two phases separatelywith different paramenters. Also if you can help me with the use of elset in this problem.

**Reply by rquey** [reply]
In -M, use something like `-rcl "(group==1)?0.5:1"`, which means "if group = 1, then rcl = 0.5, otherwise rcl = 1".

**Comment by my-name-iz-ayush** [comment]
thank you for your guidance.

**Comment by my-name-iz-ayush** [comment]
@rquey how can i get grain boundaries visible in inp file after meshing the microstructure.

**Reply by rquey** [reply]
Use [-dim 2,3](https://neper.info/doc/neper_m.html#cmdoption-dim).
